Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,13 @@
 import React from 'react';
 import './Card.css';
+
+
+const CardBanner = () => {
+	return (
+		<div>
+			<img src="https://tk-assets.lambdaschool.com/fcd75197-7d12-46ec-bc9e-4130f34822fa_reactbackground.png" alt="React Atom Logo" className="indent" />
```

Comment:
remember that one of the big factors of react is to keep the components reusable. In it's current state, what keeps this from being reusable is the fact that you hard coded data into the child. What if another user posts and their image should be different? Look into passing as much as you can down as props and just let the parent component define it later. While building the small components you just have to tell it where you want things.. the parent will tell it what it gets.